Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -117,11 +117,9 @@ Range: 2m
 Accuracy Modifier: +0
 STR Requirement: 6
 
-	Considered concussive (explosive) damage. Hits armor points 
-before physical damage if the target's armor has defense against explosive (i.e. 
-concussive) damage. If armor roll is greater than 50, target rolls a DC 50 
+	If armor roll is greater than 50, target rolls a DC 50 
 fort save for a broken limb of your choice. -20 stealth rolls for being quiet 

```

Comment:
Scope creep: needs more clarity re results of save on success vs failure
May want to specify that the "you" here is the wielder.
What even are our rules for broken limbs?

Reply:
We don't have them. This weapon is ancient.